tests/test_threshold_optimization.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from app.threshold_optimization import (ComputeThresholds, build_score_matrix,
```

Comment:
nit: rename threshold_optimization to thresholds 
and
(optional) remove 'thresholds' from function names e.g. compute(), optimize()